samples/bluetooth/broadcast_config_tool/include/broadcast_config_tool.h

```diff
@@ -121,54 +109,6 @@ static struct bap_preset bap_presets[] = {
 	{.preset = &lc3_preset_48_6_2, .name = "48_6_2"},
 };
 
-static struct audio_context contexts[] = {
-	{.context = BT_AUDIO_CONTEXT_TYPE_PROHIBITED, .name = "Prohibited"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED, .name = "Unspecified"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL, .name = "Conversational"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_MEDIA, .name = "Media"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_GAME, .name = "Game"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_INSTRUCTIONAL, .name = "Instructional"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_VOICE_ASSISTANTS, .name = "Voice assistants"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_LIVE, .name = "Live"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_SOUND_EFFECTS, .name = "Sound effects"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_NOTIFICATIONS, .name = "Notifications"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_RINGTONE, .name = "Ringtone"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_ALERTS, .name = "Alerts"},
-	{.context = BT_AUDIO_CONTEXT_TYPE_EMERGENCY_ALARM, .name = "Emergency alarm"},
-};
-
-static struct audio_location locations[] = {
```

Comment:
I had this short hand name so that people didn't have to spell out `F r o n t   L e f t` every time, we can discuss whether or not to have it there.